### PR TITLE
feat(BX-349): Add cascading end time banner to lot and sale page

### PR DESCRIPTION
**Pull request by annacarey**
This PR resolves [BX-349](https://artsyproduct.atlassian.net/browse/BID-349)

### Description

Adding cascading end time banner to the lot and sale page if the sale is open and if the sale has cascading end times. The banner shows the "Learn more" link if the env variable is configured to include the link.

Sale page:
Screen Shot 2022-04-14 at 11 10 59 AM

Lot Page:
Screen Shot 2022-04-14 at 5 14 37 PM

### Seeking feedback/help:
~~1. Because the banner is fit between two pieces of info as part of `ArtworkTombstone`, the margins on either side of the banner seem difficult to avoid but don't match the designs.
It's supposed to look like the following screenshot from figma w/ no margins:
Screen Shot 2022-04-14 at 11 07 42 AM~~
I fixed this with @dzucconi and it looks great now!

2. I chose not to use a feature flag here because the changes seems relatively low risk and if the banner doesn't work properly, it's not a huge deal. But open to other opinions.

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look on our [docs], or get in touch with us.

[app state migration]: /docs/adding_state_migrations.md
[feature flag]: /docs/developing_a_feature.md
[docs]: /docs/README.md

### CODE REVIEW

**Review by pvinis — DISMISSED** [review]

**Review by bhoggard — APPROVED** [review]

**Review by pvinis — APPROVED** [review]

File: src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          <Text
+            color="white"
+            onPress={() => navigate(helpArticleLink)}
+            style={{ textDecorationLine: "underline" }}
+          >
+            Learn more.
+          </Text>
+        )}
+      </Text>
+    </Flex>
```
Comment (olerichter00) [inline src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx:30]: I wonder if you could use Palette's [Banner](https://github.com/artsy/eigen/blob/fae0a2e2eb7078a835f9fe5c4e09e29bf5cfc74f/src/palette/elements/Banner/Banner.tsx#L16) instead 🤔

File: src/app/Scenes/Artwork/Components/CommercialInformation.tests.tsx [hunk 2]
```diff
@@ -623,7 +623,7 @@ const CommercialInformationArtworkInCascadingEndTimeAuction = {
     endAt: "2019-08-16T20:20:00+00:00",
     liveStartAt: null,
     formattedStartDateTime: "Ends Aug 16 at 8:20pm UTC",
-    cascadingEndTimeInterval: 60,
+    cascadingEndTimeIntervalMinutes: 1,
```
Comment (olerichter00) [inline src/app/Scenes/Artwork/Components/CommercialInformation.tests.tsx:626]: 👏

**Review by olerichter00 — APPROVED** [review]
Looks good 🌟 I've just added a suggestion about using Palette's already existing `Banner` component.

File: src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx (hunk 1 above)
Comment (annacarey) [reply src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx:30]: Yes I tried but I'm having trouble making it blue... the color from the v3 theme seems to be more of a light blue/purple color. It [worked on Force](https://github.com/artsy/force/blob/48f7c426f5454651c0449294d9d1815b503a4914/src/v2/Apps/Auction/Components/AuctionDetails/CascadingEndTimesBanner.tsx#L18) to wrap it in something with a blue background, but that didn't work here for some reason.

**Pull request merged**